src/components/transmuters/row/Deposit.tsx

```diff
@@ -87,6 +89,26 @@ export const Deposit = ({
     }
   }, [depositReceipt, queryClient]);
 
+  const {
+    data,
+    isError: aprQueryError,
+    isPending: aprQueryPending,
+  } = useTransmuterApr(transmuter);
+
+  // proj apr estimated roughly based on the previously fetched apr by checking the ratio of current assets to new assets and currentApr
+  // to newApr, based on user input of alAssets to add as a deposit.
+  // projectedApr / apr = assetsCurrentlyDeposited / (assetsCurrentlyDeposited + assetsToDeposit)
+  // projectedApr = apr * (assetsCurrentlyDeposited / (assetsCurrentlyDeposited + assetsToDeposit))
+  const formattedTotalAssetsDeposited = Number(
+    formatUnits(transmuter?.totalUnexchanged || BigInt(0), 18),
+  );
+  const assetsToDeposit = Number(depositAmount || 0);
```

Comment:
In the case of line 103 || makes more sense because the BigInt cast is needed, but generally speaking no real reason other than habit, since they accomplish similar end results. The extra value read is probably negligible performance wise in the case that depositAmount is 0, but you're right ?? is more strict and I have no issues switching to that if it's your preference